src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java

```diff
@@ -95,6 +95,26 @@ public void onChildViewRemoved(View parent, View child)
 		);
 
 		setClipChildren(false); // The actual clipping will be calculated in managed code
+		setClipToPadding(false); // Same as above
```

Comment:
nit: better to be explicit and redundant.
```suggestion
		setClipToPadding(false); // The actual clipping will be calculated in managed code
```